src/CotMG/ui/StaneksGiftRoot.tsx

```diff
@@ -19,7 +19,7 @@ interface IProps {
 
 export function StaneksGiftRoot({ staneksGift }: IProps): React.ReactElement {
   const rerender = useRerender();
-  useEffect(() => StaneksGiftEvents.subscribe(rerender), []);
+  useEffect(() => StaneksGiftEvents.subscribe(rerender), [rerender]);
```

Comment:
Since useRerender returns a useCallback now, don't we know that `rerender` will never change value, and there is no point in this change to the useEffect params?

Reply:
That's true, but react hooks eslint plugin does not know that and we are forced to mention `rerender` as a dependency of `useEffect`, `useCallback` or `useMemo` (unless there's a way to configure this plugin to ignore it, just like it does for example in case of `useRef` variables)

Reply:
More on this topic: https://github.com/facebook/react/issues/16873

Reply:
Ugly but I guess there's no harm in including it in the dependencies.

Reply:
Totally agree. I was looking for a solution to this, but it's just not provided in the official plugin :(